api/v1alpha1/promotion_types.go

```diff
@@ -96,6 +96,8 @@ type PromotionStatus struct {
 	// Metadata holds arbitrary metadata set by promotion mechanisms
 	// (e.g. for display purposes, or internal bookkeeping)
 	Metadata map[string]string `json:"metadata,omitempty" protobuf:"bytes,3,rep,name=metadata" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
+	// PromotedFreight is the detail of the piece of freight that was referenced by this promotion.
+	PromotedFreight *FreightReference `json:"promotedFreight,omitempty" protobuf:"bytes,4,opt,name=promotedFreight"`
```

Comment:
On the fence about the naming of this field, as it is also set when the promotion has not completed and/or failed. While the current name may make you think it's the result of a successful promotion.

Given this, I wonder if it maybe should just be named `Freight`?

cc: @krancour

Reply:
Agreed. FreightRef maybe?

Reply:
It would be fine to just call it `Freight`. I don't think there would be any ambiguity in doing so.

Reply:
Agreed.